das_client/app/lib/util/time_format.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:intl/intl.dart';
+
+class TimeFormat {
```

Comment:
Nitpick: `Format` und `TimeFormat` sind aktuell zwei sehr kleine Utility-Klassen. Ich hätte die wahrscheinlich in einem File gelassen.